src/pages/map/index.tsx

```diff
@@ -63,14 +68,23 @@ const LandmarkTabPanel = ({ latitude }: { latitude: Latitude }) => {
 };
 
 const LandmarkCard = ({ landmark }: { landmark: Landmark })  => {
-  return (
-    <Card>
-      <CardHeader>
-        <Heading size="md">{landmark.名称}</Heading>
+  const maxChar: number = useBreakpointValue({ base: mobileTextLimit, 
+    [sidebarBreakpoint]: desktopTextLimit }) || desktopTextLimit;
+  const isTruncated = landmark.定义.length > maxChar;
+  const cardText = isTruncated? <>
+  <span>{landmark.定义.substring(0, maxChar)}...</span>
+  <Link>更多</Link>
+  </>  
```

Comment:
indentation is messed up. do this:

```
const cardText = landmark.定义.length <= maxChar ? landmark.定义 : <>
  <span>{landmark.定义.substring(0, maxChar)}...{' '}</span>
  <Link>更多</Link>
</>;
```

notice the additional whitespace after `...`.

and remove isTruncated